automatic/tortoisesvn/update.ps1

```diff
@@ -0,0 +1,74 @@
+import-module au
+import-module "$PSScriptRoot/../../extensions/chocolatey-core.extension/extensions/chocolatey-core.psm1"
```

Comment:
you can change this to
```powershell
import-module "$PSScriptRoot/../../extensions/extensions.psm1"
```
just in case extension dependencies change, you won't have to add additional imports (and it's shorter 😏) .